features/tvwelcome/src/main/java/com/kape/tvwelcome/ui/vm/TvWelcomeViewModel.kt

```diff
@@ -10,8 +11,11 @@ import org.koin.core.component.KoinComponent
 
 class TvWelcomeViewModel(
     private val router: Router,
+    buildConfigProvider: BuildConfigProvider,
 ) : ViewModel(), KoinComponent {
 
+    val shouldShowSubscribeButton = buildConfigProvider.isAmazonFlavor().not()
```

Comment:
We also need to hide it for the web version. It's called `noinapp`.

Reply:
👍 
Changed it to buildConfigProvider.isGoogleFlavor() instead